api/candlepin-api-spec.yaml

```diff
@@ -10,7 +10,29 @@ servers:
 security:
   - basicAuth: []
 
-paths: {}
+paths:
+  /:
+    get:
+      description: Retrieves a list of links corresponding to Root resources
+      tags:
+        - root
+      operationId: getRootResources
+      security: []  # this resource doesn't require auth
```

Comment:
This is probably a bit late to be asking this, but what is the intent for security here? Do we just throw our existing verify annotations in here for now (where appropriate) and then change this when we get around to updating our security model?

Reply:
Yes, I think we can rework our security model in the future. Going ahead and specifying auth or noauth (by adding the line above) conveys intent in the spec, but we should keep our custom security annotations on the resource impl for now.